eo-maven-plugin/src/main/java/org/eolang/maven/PullMojo.java

```diff
@@ -89,11 +90,13 @@ public final class PullMojo extends SafeMojo {
     private Objectionary objectionary;
 
     @Override
-    public void exec() throws IOException {
+    public void exec() throws IOException, JSONException, InterruptedException {
         final Collection<Tojo> tojos = this.scopedTojos().select(
             row -> !row.exists(AssembleMojo.ATTR_EO)
-                && !row.exists(AssembleMojo.ATTR_XMIR)
+            && !row.exists(AssembleMojo.ATTR_XMIR)
         );
+        final int maxtries = 2;
```

Comment:
@yasamprom Please, read this: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html